lambdaclass/mirra_backend
# Add new layout for araban to seeds

> [!WARNING]  
> Needs to be merged along with [this client PR](https://github.com/lambdaclass/champions_of_mirra/pull/2012)

## Motivation

We wanted to add new elements to the map, to make it more interesting and add variability from match to match. This is just a first iteration that moves spikes to different positions and adds bushes that hide players and two kinds of pools, one that make characters slower and other that allows projectiles to pass through them.

## Summary of changes

Added a couple of slowing pools, lakes that only let projectiles pass through and multiple bushes.

## How to test it?

Try out in the game client, you can also use the client application to simulate multiple players in a same match, but it would be advisable to try it out with other real players. The idea is to test all new map elements and that their behavior is as expected as well as if matches feel more entertaining with these changes.

## Checklist
